### DISCUSSION TITLE
Extended creature info

### DISCUSSION THREAD

**Discussion by Branikolog** [ihhub/fheroes2#3707] [Ideas]
Let's discuss this new fascinating feature of fheroes2! I'm sure there're some points that could be improved.

**Comment by Branikolog** [comment]
https://github.com/ihhub/fheroes2/issues/3397#issuecomment-863647730
I like the most of comments @LeHerosInconnu made above.
But I think, that we shouldn't take away the "Elemental" attribute. We already have similar attribute for undead creatures, that includes no morale, morale penalty for allies, bless immunity, mind spells immunity.
Let the "Elementals" attribute include no morale penalty and mind spells immunity (Which is lacking currently). 
Yes, we can watch the "Elemental" in the name of creature. But we can add some new creatures, that could have same features in future. So adding "Elemental" attribute to them would include all these immunities and features.

**Reply by ihhub** [reply]
How about Dragons then?

**Reply by Branikolog** [reply]
Dragons, were not implemented as separate attribute (In HoMM series), just because of lacking essential and multiple features (Which could be same for different types of dragons). In H3, for example, dragons had only breath attack and magic resistance (which was partial for some dragons and full for others). Implementing such class for Heroes2 is possible, but not really necessary. (For now at least). But in future, I can consider this attribute for dragon-like creatures immune to magic and having breath attack...

**Reply by LeHerosInconnu** [reply]
Hello everyone,

I think the term "Elemental." is too close to the name of the creature, we should find something else, let's take some time. :)

**Reply by Branikolog** [reply]
What about "non-alive" then? Non-living beings don't have mind and morale.

**Reply by LeHerosInconnu** [reply]
I had also thought about "Mindless.", but I am not decided.

**Reply by zenseii** [reply]
Mindless is correct, but also means stupid which isn't the best. Some "less intelligent" creatures could fit this description too 😝

Are we only talking about elementals here? They don't have feelings and a consciousness, so how about "Non-sentient"? 

Non-living is good too and could be used in the future for additions. Both Golems should've had this, and I'm surprised they are considered having a mind when they clearly don't. In later games they're "Mechanical".

Also the game manual has a description for Water Elementals like this:
"Neutral morale.  Immune to mind and cold spells.  Fire spells do x2damage."

Personally I would always prefer to stick to the official descriptions with the occasional corrections of mistakes like "Meteor swarm".

**Comment by Branikolog** [comment]
Also, talking about Ghosts: "Soul Eater", if I'm not mistaken, was proposed by me, but it's not the best, actually, description for this feature of ghosts. "Eating" is not so clear in this case. I rather prefer "Absorbing" as ghosts not just eat souls, but absorb, making souls the part of a ghost stack.
Quite sure, other our contributors could have even more nice ideas, relating to "eating" abilities. :)

**Reply by LeHerosInconnu** [reply]
Hello everyone,

Personally I like "Soul Harvester." proposed by @zenseii.
I think it captures the "collect", "reap", "recover" side of things. :)

**Reply by zenseii** [reply]
Thanks @LeHerosInconnu  but you gave me a better idea now: "Soul Reaper".
In addition to being a good description of what happens, this gives a slight association to the Grim Reaper, who makes souls his own after they're dead.

Anyways I do agree "Eater" sounds a bit weird, and Branikolog's suggestion is good too!

**Comment by Branikolog** [comment]
I've heard another proposal from our fans regarding creatures' descriptions. 
"Flying" ability is missing currently. Not all creatures have wings and not obvious that they are able to fly. Especially ghosts and vampires.
I think we should also add this ability as there can appear some new creatures, that can fly in future.
